std/experimental/allocator/common.d

```diff
@@ -512,6 +512,18 @@ version(unittest)
         assert(b2.length == 2);
         assert(b2.ptr + b2.length <= b1.ptr || b1.ptr + b1.length <= b2.ptr);
 
+        // Test allocateZeroed
+        static if (hasMember!(A, "allocateZeroed"))
+        {{
+            auto b3 = a.allocateZeroed(8);
```

Comment:
Looks like the recent GCAllocator PR (#6432) broke this:

```
std/experimental/allocator/common.d(518): Error: `shared` mutable method `std.experimental.allocator.gc_allocator.GCAllocator.allocateZeroed` is not callable using a `shared` `const` object
std/experimental/allocator/common.d(518):        Consider adding `const` or `inout` to std.experimental.allocator.gc_allocator.GCAllocator.allocateZeroed
```

Reply:
Is this still a problem?